### DISCUSSION TITLE
Can't proceed with Github Education Student request

### DISCUSSION THREAD

**Discussion by lucascerfig** [community/community#127748] [GitHub Education]
Hi,

I am having trouble reactivating my student benefits on [GitHub Education](https://education.github.com/discount_requests/application).

Even though I have correctly set up my school emails with the requested domain, I still receive the following error:

![image](https://github.com/community/community/assets/55245275/777e8a74-217e-4b30-9c74-acd1d1b431cc)

I have also filled in all billing details, uploaded a profile photo, etc. I have been experiencing this problem for over a week and have tried different browsers, clearing cache, and other troubleshooting steps.

This is the first time I have had trouble reactivating my benefits after using GitHub Student benefits for over three years.

Any clues on how to resolve this issue?

Thank you.

**Comment by davevad93** [comment]
Hi @lucascerfig ,

Considering that you have already reapplied for your benefits in the past, it could be an internal GitHub Education problem in handling your request. I suggest you to contact support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and describe the issue with your application.
